packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -807,6 +808,36 @@ export default class Meetings extends WebexPlugin {
     );
   }
 
+  /**
+   * Creates a noise reduction effect
+   *
+   * @param {INoiseReductionEffect} options optional custom effect options
+   * @returns {Promise<effect>} noise reduction effect.
+   * @public
+   * @memberof Meetings
+   */
+  createNoiseReductionEffect = async (options?: INoiseReductionEffect) => {
+    // @ts-ignore
+    const authToken = this.webex.credentials.supertoken.access_token;
+
+    return new mediaHelpersModule.NoiseReductionEffect({authToken, ...options});
+  };
+
+  /**
+   * Creates a virtual background effect
+   *
+   * @param {Partial<IVirtualBackgroundEffect>} options optional custom effect options
+   * @returns {Promise<effect>} virtual background effect.
+   * @public
+   * @memberof Meetings
+   */
+  createVirtualBackgroundEffect = async (options?: Partial<IVirtualBackgroundEffect>) => {
```

Comment:
why is `Partial` used here but not in `createNoiseReductionEffect()`? I guess you're using partial because you don't want the authToken. I think it's better to use [typescript's omit](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys) to define the new type without the authToken

Reply:
Yes I'm already using `Omit` syntax to remove authToken but some of the properties were required in NoiseReductionEffect `constructor` apart from authToken so that's why used Partial as well. But just checked on the constructor of NoiseReductionEffect, `web-media-effects` library already using Partial so we don't have to pass.

Removing it from here then.